revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
+            width: 100%;
+            height: 40px;
+            position: absolute;
+            top: 50%;
+            display: inline-block;
+            margin: 0;
+
+            .container {
+                position: relative;
+                color: white;
+                font-weight: 500;
+                width: 100%;
+                height: 100%;
+                margin: 0;
+                display: inline-block;
+
+                .amount-donated {
+                    float: left;
+                    width: 30%;
+                    height: 100%;
+                    line-height: 40px;
+                    margin: 0px 0px 0px 20px;
+                    font-size: 1.4rem;
+                }
+
+                .days-left {
+                    float: right;
+                    height: 100%;
+                    line-height: 40px;
+                    font-size: 1.4rem;
+                    margin: 0px 20px 0px 0px;
+                }
+            }
+        }
+
+        .bottom-bar {
+            background-color: #064f7e;
+            width: 100%;
+            height: 40px;
+            position: absolute;
+            top: 60%;
+            display: inline-block;
+
+            .container {
+                position: relative;
+                color: white;
+                font-weight: 500;
+                width: 100%;
+                height: 100%;
+                margin: 0;
+
+                .amount-saved {
+                    float: left;
+                    height: 100%;
+                    line-height: 40px;
+                    font-size: 1.4rem;
+                    margin: 0px 0px 0px 20px;
+                }
+
+                .num-donors {
+                    float: right;
+                    height: 100%;
+                    line-height: 40px;
+                    font-size: 1.4rem;
+                    margin: 0px 20px 0px 0px;
+                }
+            }
+        }
+
+        .svg-graphics-container {
+            height: 100%;
+            width: 100%;
+            position: absolute;
+        }
+
+        .internal-graphics-container {
+            height: 220px;
+            width: 220px;
+            position: relative;
+            left: 50%;
+            top: 50%;
+            -webkit-transform: translate(-50%, -50%);
+            -ms-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
+            position: absolute;
+        }
+
+        .project-badge-line, .project-badge-circle {
+          fill: none;
+          stroke-width: 20px;
+        }
+
+        .project-badge-line {
+          stroke: $revolv-blue;
+        }
+
+        .project-badge-circle {
+          stroke: white;
+        }
+
+        .text-container {
+            position: absolute;
+            left: 50%;
+            top: 50%;
+            -webkit-transform: translate(-50%, -50%);
+            -ms-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
```

Comment:
Why do you need all this translation? They won't work in IE 8 ([http://caniuse.com/#feat=transforms2d](http://caniuse.com/#feat=transforms2d))